eve/main.yml

```diff
@@ -562,3 +562,4 @@ stages:
           name: push dashboards to the production namespace
           command: |
             oras push ${REGISTRY}/${PROJECT}-dev/${PROJECT}-dashboards:$revision ${LAYERS}
+          env: *oras
```

Comment:
don't we also need `workdir: build/monitoring/`, like line 522 ?

Reply:
that might be right :laughing: